### DISCUSSION TITLE
Add podcast locally

### DISCUSSION THREAD

**Discussion by Vctrsnts** [SamTV12345/PodFetch#1021] [General]
Hi.
Is it possible to create a directory with the name of a podcast, copy `mp3 files` of the podcast that I downloaded from another site and have **podfetch** recognize that directory and files as a new podcast?
Or can it only be used by adding `rss`?
Thanks

**Comment by SamTV12345** [comment]
Currently only RSS. You could spin up an nginx container with your rss feed and the files. I'll build a podcast manager. But it will probably take a while til this is implemented.

**Comment by Vctrsnts** [comment]
Thank you very much for the work you do.
Don't worry, if I've been using it as I am right now ( audiobookshelf ) I can wait longer.
Although I don't like wasting my server's resources for a process, which I understand can be easier.
Again, thanks for everything
